tests/src/test_projects/signed_i128/tests/mod.rs

```diff
@@ -16,6 +16,8 @@ mod success {
 
         let instance = Testi128::new(wallet, path_to_bin);
 
-        let _result = instance.main().call().await;
+        let params = TxParameters::new(Some(1), Some(10000000), None);

```

Comment:
Same suggestion for other instances of this, for readability purposes.

```suggestion
        let params = TxParameters::new(Some(1), Some(10_000_000), None);

```